### ISSUE TITLE
Can not apply plugin in configure block

### ISSUE THREAD

**Issue by JLLeitschuh** [gradle/gradle-script-kotlin#224]
```kotlin
configure(subprojects - listOf(project(":PlexxiCorePythonAPI"), project(":PlexxiInstall"))) {
    apply {
        plugin("java") // This shows up as an error in intellij
    }
}
```
<img width="941" alt="screen shot 2017-01-16 at 11 42 28 am" src="https://cloud.githubusercontent.com/assets/1323708/21991520/f209bbc6-dbe0-11e6-8187-948b2360dfe6.png">

The current workaround is to do this:
```kotlin
configure(subprojects - listOf(project(":PlexxiCorePythonAPI"), project(":PlexxiInstall"))) {
    apply(delegateClosureOf<ObjectConfigurationAction> {
        plugin("java")
    })
}
```

**Comment by bamboo** [comment]
Blocked by #157

**Comment by bamboo** [comment]
The original code is correct and works even though IntelliJ marks it as an error.

**Comment by bamboo** [comment]
Resolved in a7308886672047171bcddcb9979c0a22295f925f.
